### PR TITLE
Rename project to zeebe-process-test

### PR DESCRIPTION
**Pull request by remcowesterhoud**
## Description

I recommend reviewing by commit

We have decided on a definitive name for the project. We went with `zeebe-process-test`. This is because it does not contain BPMN, for when we want to support DMN in the future. It also doesn't contain assertion which caused confusion, as the project provided more than just assertions.

- Renamed the artifact (also in README)
- Renamed the scm settings
- Renamed the modules

This repository has also been renamed. This means your git remote url should also be updated!

We still need to rename some stuff in the GitHub workflows. This will be done in a later PR as this needs te done together with the infra team.

## Definition of Done

_Not all items need to be done depending on the issue and the pull request._

Code changes:
* [ ] The changes are backwards compatibility with previous versions
* [ ] If it fixes a bug then PRs are created to backport the fix

Testing:
* [x] There are unit/integration tests that verify all acceptance criterias of the issue
* [ ] New tests are written to ensure backwards compatibility with further versions
* [ ] The behavior is tested manually

Documentation:
* [ ] Javadoc has been written
* [x] The documentation is updated

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Add the following dependency to your project
 ```xml
 <dependency>
   <groupId>io.camunda</groupId>
-  <artifactId>zeebe-bpmn-assert</artifactId>
+  <artifactId>zeebe-process-test</artifactId>
```
Comment (pihme) [inline README.md:19]: :x: Please also consider renaming top level heading in Readme. And maybe also rephrasing the first paragraph Camunda Cloud vs. Zeebe. 

:x: There is also this line in the readme.md which needs to be updated
> For example tests the best place to look right now is the tests in io.camunda.zeebe.bpmnassert.assertions

And maybe do a general text search for the old name to see if anything else remained

**Review by pihme — CHANGES_REQUESTED** [review]
Looks good. Some details are missing.

File: README.md (hunk 1 above)
Comment (pihme) [reply README.md:19]: :clown_face: Love the avatar, by the way :clown_face:

Comment (remcowesterhoud) [reply README.md:19]: Thanks 😄 

I've fixed the README. The general text search yielded no results apart from the Vault which I mentioned before. That'll probably be fixed in a separate PR tomorrow.

**Review by pihme — APPROVED** [review]

**Pull request merged**